Weather App/controller/SearchTabViewController.swift

```diff
@@ -0,0 +1,89 @@
+import SnapKit
+import UIKit
+
+class SearchTabViewController: UIViewController {
+    private var searchTextField: UISearchBar!
+    private var tableView: UITableView!
+    private let cities = ["Łódź", "Warszawa", "Kraków", "Wrocław", "Gdańsk", "Gdynia", "Bydgoszcz", "Sopot", "Kielce"]
```

Comment:
it will be better to store cities in some provider, which give you data source.

Reply:
👍